README.md

```diff
@@ -25,6 +25,26 @@ Adminer:
 localhost:9000
 ```
 
+<<<<<<< HEAD
```

Comment:
This is merge conflict marker which you had to remove during conflict resolving.